src/libraries/JANA/Services/JParameterManager.h

```diff
@@ -383,13 +414,40 @@ inline void JParameterManager::Parse(const std::string& value, std::vector<T> &v
     std::stringstream ss(value);
     std::string s;
     val.clear(); // clearing the input vector to ensure no dulication which can be caused due to val.push_back(t);
+    std::string temp = "";   // creating a temp var allows us to store the string s where the escape character was used
     while (getline(ss, s, ',')) {
         T t;
         Parse(s, t);
         val.push_back(t);
     }
 }
 
+/// @brief Specialization for std::vector<std::string> with escape commas
+template<>
+inline void JParameterManager::Parse(const std::string& value, std::vector<std::string> &val) {
+    std::stringstream ss(value);
+    std::string s;
+    val.clear(); // clearing the input vector to ensure no dulication which can be caused due to val.push_back(t);
+    std::string temp = "";   // creating a temp var allows us to store the string s where the escape character was used
+    while (getline(ss, s, ',')) {
+        std::string t;
+        if (s.find('\\') != std::string::npos) {
```

Comment:
This finds and removes the first escape character anywhere in the string, not necessarily at the end. I'd replace this with simply checking whether the _last_ chararacter is a `\\`, which is much faster, anyhow.

Test case: `"All of this belongs in\t slot one\, including the tab character,whereas all of this belongs in slot two"`